src/scss/layout/_footer.scss

```diff
@@ -216,6 +217,12 @@ footer {
 				svg {
 					margin-bottom: rem(16);
 				}
+
+				.hewlett-logo {
+					height: rem(80);
+					padding: rem(13) rem(13) 0 0;
```

Comment:
This is a duplicate line as what has been set for `.hewlett-logo` in the mobile layout. It can be removed.